code/game/atoms_movable.dm

```diff
@@ -531,17 +531,19 @@
 	return
 
 
-/atom/movable/proc/do_attack_animation(atom/A, visual_effect_icon, obj/item/used_item, no_effect, end_pixel_y)
+/atom/movable/proc/do_attack_animation(atom/A, visual_effect_icon, obj/item/used_item, no_effect, end_pixel_y, final_pixel_y, final_pixel_x)

```

Comment:
// CITADEL CHANGE - adds final_pixel_y and final_pixel_x args to do_attack_animation